translations/strings.yaml

```diff
@@ -274,12 +274,12 @@ networkSettings:
 
 captivePortalAlert:
   title: Captive portal detected!
-  header: A captive portal is when the guest Wi-Fi you are using requires you to log in to get internet access. 
```

Comment:
Can we enforce no trailing whitespaces via linter? It's really not useful to blame a file, and I'd prefer to enforce this.

Reply:
Yes. Let's do this. follow-up.